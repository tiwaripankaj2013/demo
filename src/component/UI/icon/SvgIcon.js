import React from 'react';

const SvgIcon = ( WrappedComponent) => {
   return class SvgIcon extends React.Component {
      render() {
         let {style,onClick} = this.props;
         const _Style = {...style}
        return(
         <span className='icon' style={ _Style} onClick={ onClick}>
           <WrappedComponent {...this.props} />
         </span>
        );
      }
    }
}

export default SvgIcon;