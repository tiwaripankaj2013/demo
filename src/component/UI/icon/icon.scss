.storyIcon{
   .icon{margin-right: 0.5rem;}
}
.icon{vertical-align: middle;
   display: inline-block;
   line-height: 0;}