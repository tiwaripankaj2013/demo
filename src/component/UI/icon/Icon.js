import React from 'react';
import SvgIcon from './SvgIcon';
export const ChartIcon = SvgIcon(() => {
   return (<svg width="1em" height="0.904em" viewBox="0 0 42 38" fill="none" xmlns="http://www.w3.org/2000/svg">
   <path d="M3 22.3271C3 21.2226 3.89543 20.3271 5 20.3271C6.10457 20.3271 7 21.2226 7 22.3271V29.4271C7 30.5317 6.10457 31.4271 5 31.4271C3.89543 31.4271 3 30.5317 3 29.4271V22.3271Z" fill="currentColor"/>
   <path d="M11 14.9272C11 13.8227 11.8954 12.9272 13 12.9272C14.1046 12.9272 15 13.8227 15 14.9272V29.4272C15 30.5318 14.1046 31.4272 13 31.4272C11.8954 31.4272 11 30.5318 11 29.4272V14.9272Z" fill="currentColor"/>
   <path d="M19 22.3271C19 21.2226 19.8954 20.3271 21 20.3271C22.1046 20.3271 23 21.2226 23 22.3271V29.4271C23 30.5317 22.1046 31.4271 21 31.4271C19.8954 31.4271 19 30.5317 19 29.4271V22.3271Z" fill="currentColor"/>
   <path d="M27 11.227C27 10.1225 27.8954 9.22705 29 9.22705C30.1046 9.22705 31 10.1225 31 11.2271V29.4271C31 30.5316 30.1046 31.4271 29 31.4271C27.8954 31.4271 27 30.5316 27 29.4271V11.227Z" fill="currentColor"/>
   <path d="M35 11.227C35 10.1225 35.8954 9.22705 37 9.22705C38.1046 9.22705 39 10.1225 39 11.2271V29.4271C39 30.5316 38.1046 31.4271 37 31.4271C35.8954 31.4271 35 30.5316 35 29.4271V11.227Z" fill="currentColor"/>
   </svg>   
   )
});
export const TaskIcon = SvgIcon(() => {
   return (<svg width="1em" height="1em" viewBox="0 0 56 56" fill="none" xmlns="http://www.w3.org/2000/svg">
   <path d="M27.9999 21.4165C31.6016 21.4165 34.5833 24.3982 34.5833 27.9998C34.5833 31.6015 31.6016 34.5832 27.9999 34.5832C24.3983 34.5832 21.4166 31.6015 21.4166 27.9998C21.4166 24.2352 24.3865 21.4165 27.9999 21.4165Z" stroke="white" strokeWidth="1.5"/>
   <path d="M28 15.1309C35.0438 15.1309 40.8691 20.9561 40.8691 27.9999C40.8691 35.0438 35.0438 40.869 28 40.869C20.9561 40.869 15.131 35.0438 15.131 27.9999C15.131 20.6434 20.9444 15.1309 28 15.1309Z" stroke="white" strokeWidth="1.5"/>
   <path d="M27.7381 15.8775C27.3453 15.8775 26.9524 15.3184 26.9524 14.7593V7.11821C26.9524 6.55911 27.3453 6 27.7381 6C28.131 6 28.5238 6.55911 28.5238 7.11821V14.9457C28.5238 15.5048 28.2619 15.8775 27.7381 15.8775Z" fill="currentColor"/>
   <path d="M27.7381 50.0001C27.3453 50.0001 26.9524 49.4513 26.9524 48.9026V41.2201C26.9524 40.6713 27.3453 40.1226 27.7381 40.1226C28.131 40.1226 28.5238 40.6713 28.5238 41.2201V48.9026C28.5238 49.6343 28.2619 50.0001 27.7381 50.0001Z" fill="currentColor"/>
   <path d="M14.7801 28.5238H7.09751C6.54876 28.5238 6 28.131 6 27.7381C6 27.3453 6.54876 26.9524 7.09751 26.9524H14.7801C15.3288 26.9524 15.8776 27.3453 15.8776 27.7381C15.6946 28.2619 15.3288 28.5238 14.7801 28.5238Z" fill="currentColor"/>
   <path d="M48.9025 28.5238H41.2199C40.6711 28.5238 40.1224 28.131 40.1224 27.7381C40.1224 27.3453 40.6711 26.9524 41.2199 26.9524H48.9025C49.4512 26.9524 50 27.3453 50 27.7381C50 28.2619 49.6342 28.5238 48.9025 28.5238Z" fill="currentColor"/>
   </svg>   
   )
});
export const GraphIcon = SvgIcon(() => {
   return (<svg width="1em" height="1em" viewBox="0 0 56 56" fill="none" xmlns="http://www.w3.org/2000/svg">
   <path d="M28.1373 26.4435L10.6868 19.2201C10.2746 19.0956 10.1371 18.5974 10.2745 18.3484C10.412 17.9747 10.9616 17.8502 11.2364 17.9747L27.5876 24.8245L42.1526 11.4986C42.4275 11.2495 42.9771 11.2495 43.2519 11.4986C43.5267 11.7477 43.5267 12.2458 43.2519 12.4949L28.1373 26.4435Z" fill="currentColor"/>
   <path d="M43.252 17.4762C42.8398 17.4762 42.4275 17.1025 42.4275 16.7289V12.4945H37.4809C37.0687 12.4945 36.6565 12.1209 36.6565 11.7472C36.6565 11.3736 37.0687 11 37.4809 11H43.252C43.6642 11 44.0764 11.3736 44.0764 11.7472V16.7289C43.939 17.2271 43.6642 17.4762 43.252 17.4762Z" fill="currentColor"/>
   <path d="M14.5344 27.9378C12.8855 27.9378 11.5115 29.1832 11.5115 30.6777V40.8901C11.5115 42.3846 12.8855 43.63 14.5344 43.63C16.1832 43.63 17.5573 42.3846 17.5573 40.8901V30.6777C17.5573 29.1832 16.3206 27.9378 14.5344 27.9378ZM14.5344 45C12.0611 45 10 43.1319 10 40.8901V30.6777C10 28.436 12.0611 26.5679 14.5344 26.5679C17.0077 26.5679 19.0688 28.436 19.0688 30.6777V40.8901C19.2062 43.1319 17.1451 45 14.5344 45Z" fill="currentColor"/>
   <path d="M28.0002 31.4246C26.3513 31.4246 24.9773 32.6701 24.9773 34.1646V40.8898C24.9773 42.3843 26.3513 43.6297 28.0002 43.6297C29.6491 43.6297 31.0231 42.3843 31.0231 40.8898V34.1646C31.0231 32.6701 29.6491 31.4246 28.0002 31.4246ZM28.0002 44.9996C25.5269 44.9996 23.4658 43.1315 23.4658 40.8898V34.1646C23.4658 31.9228 25.5269 30.0547 28.0002 30.0547C30.4735 30.0547 32.5346 31.9228 32.5346 34.1646V40.8898C32.5346 43.1315 30.4735 44.9996 28.0002 44.9996Z" fill="currentColor"/>
   <path d="M41.4656 24.575C39.8167 24.575 38.4427 25.8204 38.4427 27.3149V40.8899C38.4427 42.3844 39.8167 43.6299 41.4656 43.6299C43.1145 43.6299 44.4885 42.3844 44.4885 40.8899V27.3149C44.4885 25.8204 43.1145 24.575 41.4656 24.575ZM41.4656 44.9998C38.9923 44.9998 36.9312 43.1317 36.9312 40.8899V27.3149C36.9312 25.0732 38.9923 23.2051 41.4656 23.2051C43.9389 23.2051 46 25.0732 46 27.3149V40.8899C46 43.1317 43.9389 44.9998 41.4656 44.9998Z" fill="currentColor"/>
   </svg>    
   )
});
export const EmailIcon = SvgIcon(() => {
   return (<svg width="1em" height="0.719em" viewBox="0 0 24 19" fill="none" xmlns="http://www.w3.org/2000/svg">
   <path d="M21 0.0654297H2.33333C1.05 0.0654297 0.0116667 1.11543 0.0116667 2.39876L0 16.3988C0 17.6821 1.05 18.7321 2.33333 18.7321H21C22.2833 18.7321 23.3333 17.6821 23.3333 16.3988V2.39876C23.3333 1.11543 22.2833 0.0654297 21 0.0654297ZM21 16.3988H2.33333V4.7321L11.6667 10.5654L21 4.7321V16.3988ZM11.6667 8.2321L2.33333 2.39876H21L11.6667 8.2321Z" fill="currentColor"/>
   </svg>    
   )
});
export const HelpCircleIcon = SvgIcon(() => {
   return (<svg width="1em" height="1em" viewBox="0 0 22 22" fill="none" xmlns="http://www.w3.org/2000/svg">
   <path d="M9.91663 17.5001H12.0833V15.3334H9.91663V17.5001ZM11 0.166748C5.01996 0.166748 0.166626 5.02008 0.166626 11.0001C0.166626 16.9801 5.01996 21.8334 11 21.8334C16.98 21.8334 21.8333 16.9801 21.8333 11.0001C21.8333 5.02008 16.98 0.166748 11 0.166748ZM11 19.6667C6.22246 19.6667 2.33329 15.7776 2.33329 11.0001C2.33329 6.22258 6.22246 2.33341 11 2.33341C15.7775 2.33341 19.6666 6.22258 19.6666 11.0001C19.6666 15.7776 15.7775 19.6667 11 19.6667ZM11 4.50008C8.60579 4.50008 6.66663 6.43925 6.66663 8.83342H8.83329C8.83329 7.64175 9.80829 6.66675 11 6.66675C12.1916 6.66675 13.1666 7.64175 13.1666 8.83342C13.1666 11.0001 9.91663 10.7292 9.91663 14.2501H12.0833C12.0833 11.8126 15.3333 11.5417 15.3333 8.83342C15.3333 6.43925 13.3941 4.50008 11 4.50008Z" fill="currentColor"/>
   </svg>
   )
});
export const ArrowUpIcon = SvgIcon(() => {
   return (<svg width="1em" height="1em" viewBox="0 0 25 25" fill="none" xmlns="http://www.w3.org/2000/svg">
   <path d="M4.52985 12.061L5.93985 13.471L11.5298 7.89104V20.061H13.5298V7.89104L19.1098 13.481L20.5298 12.061L12.5298 4.06104L4.52985 12.061Z" fill="currentColor"/>
   </svg>   
   )
});
export const ArrowDownIcon = SvgIcon(() =>{
   return(
      <svg width="1em" height="1em" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M16.59 8.59009L12 13.1701L7.41 8.59009L6 10.0001L12 16.0001L18 10.0001L16.59 8.59009Z" fill="currentColor"/>
</svg>
   )
})
export const SearchIcon = SvgIcon(() =>{
   return(
      <svg width="1em" height="1.0416em"  viewBox="0 0 20 21" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M14.2939 13.0786H13.3905L13.0703 12.7699C14.191 11.4663 14.8656 9.77387 14.8656 7.93282C14.8656 3.82762 11.538 0.5 7.43282 0.5C3.32762 0.5 0 3.82762 0 7.93282C0 12.038 3.32762 15.3656 7.43282 15.3656C9.27387 15.3656 10.9663 14.691 12.2699 13.5703L12.5786 13.8905V14.7939L18.2962 20.5L20 18.7962L14.2939 13.0786ZM7.43282 13.0786C4.58548 13.0786 2.28702 10.7802 2.28702 7.93282C2.28702 5.08548 4.58548 2.78702 7.43282 2.78702C10.2802 2.78702 12.5786 5.08548 12.5786 7.93282C12.5786 10.7802 10.2802 13.0786 7.43282 13.0786Z" fill="currentColor"/>
</svg>
   )
})
export const MenuIcon = SvgIcon(() =>{
   return(<svg width="1em" height="1.0416em" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
   <path d="M3 18.5H21V16.5H3V18.5ZM3 13.5H21V11.5H3V13.5ZM3 6.5V8.5H21V6.5H3Z" fill="currentColor"/>
   </svg>
   )
})
export const ProductIcon = SvgIcon(() =>{
   return(<svg width="1em" height="1em" viewBox="0 0 56 56" fill="none" xmlns="http://www.w3.org/2000/svg">
   <path d="M27.9989 21.1145C20.0728 21.1145 13.6146 27.467 13.6146 35.2632C13.6146 43.0594 20.0728 49.4119 27.9989 49.4119C35.925 49.4119 42.3832 43.0594 42.3832 35.2632C42.3832 27.467 35.925 21.1145 27.9989 21.1145ZM27.9989 51C19.1922 51 12 43.9257 12 35.2632C12 26.6007 19.1922 19.5264 27.9989 19.5264C36.8056 19.5264 43.9978 26.6007 43.9978 35.2632C44.1446 43.9257 36.9524 51 27.9989 51Z" fill="currentColor"/>
   <path d="M27.5898 16.2983C27.1795 16.2983 26.7693 15.8793 26.7693 15.4603V9.87321C26.7693 9.45418 27.1795 9.03516 27.5898 9.03516C28.0001 9.03516 28.4103 9.45418 28.4103 9.87321V15.5999C28.4103 16.019 28.0001 16.2983 27.5898 16.2983Z" fill="currentColor"/>
   <path d="M39.8543 6.65707C35.9245 6.95836 32.6964 10.2725 32.556 14.4905C36.4859 14.3399 39.7139 11.0257 39.8543 6.65707ZM32.135 16.2982H31.7139C31.2929 16.2982 30.8718 15.8463 30.8718 15.3944V15.0931C30.8718 9.51929 35.0824 5 40.2753 5H40.6964C41.1174 5 41.5385 5.45193 41.5385 5.90386V6.35579C41.5385 11.7789 37.328 16.2982 32.135 16.2982Z" fill="currentColor"/>
   </svg>
   
   )
})
export const MoreIcon = SvgIcon(() =>{
   return(<svg width="1em" height="1.041em" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
   <path d="M6 10.5684C4.9 10.5684 4 11.4684 4 12.5684C4 13.6684 4.9 14.5684 6 14.5684C7.1 14.5684 8 13.6684 8 12.5684C8 11.4684 7.1 10.5684 6 10.5684ZM18 10.5684C16.9 10.5684 16 11.4684 16 12.5684C16 13.6684 16.9 14.5684 18 14.5684C19.1 14.5684 20 13.6684 20 12.5684C20 11.4684 19.1 10.5684 18 10.5684ZM12 10.5684C10.9 10.5684 10 11.4684 10 12.5684C10 13.6684 10.9 14.5684 12 14.5684C13.1 14.5684 14 13.6684 14 12.5684C14 11.4684 13.1 10.5684 12 10.5684Z" fill="currentColor"/>
   </svg>
   )
})
export const PerformanceIcon = SvgIcon(() =>{
   return(<svg width="1em" height="1em" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
   <path d="M21 9.625L23.2807 7.34431M0.875 16.625H4.375H0.875ZM14 3.5V7V3.5ZM23.625 16.625H27.125H23.625Z" stroke="currentColor" strokeMiterlimit="10" strokeLinecap="round" strokeLinejoin="round"/>
   <path d="M6.64256 9.86307C6.5926 9.79357 6.5647 9.71064 6.5625 9.62507C6.56245 9.54871 6.58237 9.47368 6.62031 9.40741C6.65824 9.34114 6.71285 9.28596 6.77872 9.24734C6.84459 9.20872 6.91942 9.18801 6.99577 9.18728C7.07212 9.18654 7.14733 9.20579 7.21394 9.24313L15.4162 14.4148C15.743 14.624 16.0194 14.903 16.2255 15.2318C16.4316 15.5606 16.5622 15.931 16.6079 16.3163C16.6536 16.7017 16.6133 17.0923 16.4899 17.4602C16.3665 17.8281 16.163 18.1641 15.8942 18.4439C15.6254 18.7237 15.2978 18.9404 14.9352 19.0784C14.5725 19.2165 14.1838 19.2724 13.7969 19.2421C13.4101 19.2118 13.0347 19.0962 12.6979 18.9034C12.3612 18.7107 12.0713 18.4457 11.8493 18.1274L6.64256 9.86307Z" stroke="currentColor" strokeMiterlimit="10" strokeLinecap="round" strokeLinejoin="round"/>
   <path d="M25.1003 23.6249C26.353 21.6395 27.0502 19.3543 27.1193 17.0077C27.1884 14.6611 26.6269 12.339 25.4932 10.2832C24.3595 8.22747 22.6952 6.51342 20.6737 5.31971C18.6523 4.12601 16.3476 3.49634 14 3.49634C11.6524 3.49634 9.34774 4.12601 7.32626 5.31971C5.30478 6.51342 3.64047 8.22747 2.5068 10.2832C1.37312 12.339 0.811576 14.6611 0.88069 17.0077C0.949805 19.3543 1.64705 21.6395 2.89975 23.6249H25.1003Z" stroke="currentColor" strokeMiterlimit="10" strokeLinecap="round" strokeLinejoin="round"/>
   </svg>
   )
})
export const PlayIcon = SvgIcon(() =>{
   return(<svg width="1em" height="1em" viewBox="0 0 65 64" fill="none" xmlns="http://www.w3.org/2000/svg">
   <path d="M32.5 2C26.5666 2 20.7664 3.75947 15.8329 7.05591C10.8994 10.3524 7.05426 15.0377 4.78363 20.5195C2.513 26.0013 1.9189 32.0333 3.07645 37.8527C4.23401 43.6721 7.09123 49.0176 11.2868 53.2132C15.4824 57.4088 20.8279 60.266 26.6473 61.4236C32.4667 62.5811 38.4987 61.987 43.9805 59.7164C49.4623 57.4458 54.1477 53.6006 57.4441 48.6671C60.7405 43.7336 62.5 37.9334 62.5 32C62.5 24.0435 59.3393 16.4129 53.7132 10.7868C48.0871 5.1607 40.4565 2 32.5 2ZM24.5 46V18L48.5 32L24.5 46Z" fill="currentColor"/>
   </svg>
   )
})
export const AngleRightIcon = SvgIcon(() =>{
   return(<svg width="1em" height="1em" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
   <path d="M9.75 7.5022L15 11.9999L9.75 16.5022" stroke="currentColor" strokeWidth="2" strokeMiterlimit="10" strokeLinecap="round" strokeLinejoin="round"/>
   </svg>   
   )
})