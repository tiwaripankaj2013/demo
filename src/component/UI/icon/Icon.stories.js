import React from 'react';
import {ChartIcon,TaskIcon,GraphIcon,EmailIcon,HelpCircleIcon,ArrowUpIcon,ArrowDownIcon,MenuIcon, SearchIcon, MoreIcon} from './Icon';
import './icon.scss'
export default {
   title: 'Icons',
    argTypes: {
    fontSize: { control: { type: 'inline-radio', options: ['48px', '24px', '16px'] } },
    color: { control: 'color' },
  },
 }
 export const Icons = (args) => {
   return (
     <div  className="storyIcon" style={{ ...args }}>
        <ChartIcon />
        <TaskIcon/>
        <GraphIcon/>
        <EmailIcon/>
        <HelpCircleIcon/>
        <ArrowUpIcon/>
        <ArrowDownIcon/>
        <MenuIcon />
        <SearchIcon/>
        <MoreIcon/>
      </div>
  );
};
