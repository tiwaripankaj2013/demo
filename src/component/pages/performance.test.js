import React from 'react';
import {render} from '@testing-library/react';
import Performance from './Performance';
import {PerformanceIcon} from '../UI/icon/Icon';

describe('Performance card pass heading and subtitle and 2 tab data in array list', () => {  
    const tabData =[
        {title:"Yesterday",
        content:"content",},
        {title:"Last Week",
        content:"last week content",}
    ]
    test('it shouldrender correctly  snapshots test ',() => {
        const PerformanceCard = render(<Performance 
            logo={<PerformanceIcon/>}
            tabContent={tabData} 
            header="Performance" 
            subHeader="May 15" 
            link="https://www.figma.com/"
            linkName="More" />)
        expect(PerformanceCard).toMatchSnapshot();
        });  
});