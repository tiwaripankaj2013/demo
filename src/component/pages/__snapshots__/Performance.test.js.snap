// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Performance card it shouldrender correctly  snapshots test 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="card undefined"
      >
        <div
          class="tab"
        >
          <div
            class="tabHeader"
          >
            <div
              class="tabHeading"
            >
              <button
                class="leaf bg_brand text_white"
              >
                <span
                  class="icon"
                >
                  <svg
                    fill="none"
                    height="1em"
                    viewBox="0 0 28 28"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M21 9.625L23.2807 7.34431M0.875 16.625H4.375H0.875ZM14 3.5V7V3.5ZM23.625 16.625H27.125H23.625Z"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-miterlimit="10"
                    />
                    <path
                      d="M6.64256 9.86307C6.5926 9.79357 6.5647 9.71064 6.5625 9.62507C6.56245 9.54871 6.58237 9.47368 6.62031 9.40741C6.65824 9.34114 6.71285 9.28596 6.77872 9.24734C6.84459 9.20872 6.91942 9.18801 6.99577 9.18728C7.07212 9.18654 7.14733 9.20579 7.21394 9.24313L15.4162 14.4148C15.743 14.624 16.0194 14.903 16.2255 15.2318C16.4316 15.5606 16.5622 15.931 16.6079 16.3163C16.6536 16.7017 16.6133 17.0923 16.4899 17.4602C16.3665 17.8281 16.163 18.1641 15.8942 18.4439C15.6254 18.7237 15.2978 18.9404 14.9352 19.0784C14.5725 19.2165 14.1838 19.2724 13.7969 19.2421C13.4101 19.2118 13.0347 19.0962 12.6979 18.9034C12.3612 18.7107 12.0713 18.4457 11.8493 18.1274L6.64256 9.86307Z"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-miterlimit="10"
                    />
                    <path
                      d="M25.1003 23.6249C26.353 21.6395 27.0502 19.3543 27.1193 17.0077C27.1884 14.6611 26.6269 12.339 25.4932 10.2832C24.3595 8.22747 22.6952 6.51342 20.6737 5.31971C18.6523 4.12601 16.3476 3.49634 14 3.49634C11.6524 3.49634 9.34774 4.12601 7.32626 5.31971C5.30478 6.51342 3.64047 8.22747 2.5068 10.2832C1.37312 12.339 0.811576 14.6611 0.88069 17.0077C0.949805 19.3543 1.64705 21.6395 2.89975 23.6249H25.1003Z"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-miterlimit="10"
                    />
                  </svg>
                </span>
              </button>
              <div
                class="titleWrapper"
              >
                <h3
                  class="h3 text_black2"
                >
                  Performance
                </h3>
                <span
                  class="subtittle"
                >
                  May 15
                </span>
              </div>
            </div>
            <ul
              class="tabMenu"
            >
              <li
                class="tabMenuItem active"
              >
                <span
                  class="label"
                >
                  Yesterday
                </span>
              </li>
              <li
                class="tabMenuItem "
              >
                <span
                  class="label"
                >
                  Last Week
                </span>
              </li>
            </ul>
          </div>
          <div
            class="bg_yellow w_100 undefined"
            style="min-height: 50vh;"
          >
            <p
              class="text_center text_white m_0"
            >
              content
               
            </p>
          </div>
        </div>
        <div
          class="tabFooter text_right"
        >
          <a
            class="more_btn"
            href="/https://www.figma.com/"
            rel="noreferrer noopener"
            target="_blank"
          >
            <span>
              More
            </span>
            <span
              class="icon"
            >
              <svg
                fill="none"
                height="1em"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M9.75 7.5022L15 11.9999L9.75 16.5022"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-miterlimit="10"
                  stroke-width="2"
                />
              </svg>
            </span>
          </a>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="card undefined"
    >
      <div
        class="tab"
      >
        <div
          class="tabHeader"
        >
          <div
            class="tabHeading"
          >
            <button
              class="leaf bg_brand text_white"
            >
              <span
                class="icon"
              >
                <svg
                  fill="none"
                  height="1em"
                  viewBox="0 0 28 28"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M21 9.625L23.2807 7.34431M0.875 16.625H4.375H0.875ZM14 3.5V7V3.5ZM23.625 16.625H27.125H23.625Z"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-miterlimit="10"
                  />
                  <path
                    d="M6.64256 9.86307C6.5926 9.79357 6.5647 9.71064 6.5625 9.62507C6.56245 9.54871 6.58237 9.47368 6.62031 9.40741C6.65824 9.34114 6.71285 9.28596 6.77872 9.24734C6.84459 9.20872 6.91942 9.18801 6.99577 9.18728C7.07212 9.18654 7.14733 9.20579 7.21394 9.24313L15.4162 14.4148C15.743 14.624 16.0194 14.903 16.2255 15.2318C16.4316 15.5606 16.5622 15.931 16.6079 16.3163C16.6536 16.7017 16.6133 17.0923 16.4899 17.4602C16.3665 17.8281 16.163 18.1641 15.8942 18.4439C15.6254 18.7237 15.2978 18.9404 14.9352 19.0784C14.5725 19.2165 14.1838 19.2724 13.7969 19.2421C13.4101 19.2118 13.0347 19.0962 12.6979 18.9034C12.3612 18.7107 12.0713 18.4457 11.8493 18.1274L6.64256 9.86307Z"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-miterlimit="10"
                  />
                  <path
                    d="M25.1003 23.6249C26.353 21.6395 27.0502 19.3543 27.1193 17.0077C27.1884 14.6611 26.6269 12.339 25.4932 10.2832C24.3595 8.22747 22.6952 6.51342 20.6737 5.31971C18.6523 4.12601 16.3476 3.49634 14 3.49634C11.6524 3.49634 9.34774 4.12601 7.32626 5.31971C5.30478 6.51342 3.64047 8.22747 2.5068 10.2832C1.37312 12.339 0.811576 14.6611 0.88069 17.0077C0.949805 19.3543 1.64705 21.6395 2.89975 23.6249H25.1003Z"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-miterlimit="10"
                  />
                </svg>
              </span>
            </button>
            <div
              class="titleWrapper"
            >
              <h3
                class="h3 text_black2"
              >
                Performance
              </h3>
              <span
                class="subtittle"
              >
                May 15
              </span>
            </div>
          </div>
          <ul
            class="tabMenu"
          >
            <li
              class="tabMenuItem active"
            >
              <span
                class="label"
              >
                Yesterday
              </span>
            </li>
            <li
              class="tabMenuItem "
            >
              <span
                class="label"
              >
                Last Week
              </span>
            </li>
          </ul>
        </div>
        <div
          class="bg_yellow w_100 undefined"
          style="min-height: 50vh;"
        >
          <p
            class="text_center text_white m_0"
          >
            content
             
          </p>
        </div>
      </div>
      <div
        class="tabFooter text_right"
      >
        <a
          class="more_btn"
          href="/https://www.figma.com/"
          rel="noreferrer noopener"
          target="_blank"
        >
          <span>
            More
          </span>
          <span
            class="icon"
          >
            <svg
              fill="none"
              height="1em"
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M9.75 7.5022L15 11.9999L9.75 16.5022"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-miterlimit="10"
                stroke-width="2"
              />
            </svg>
          </span>
        </a>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
